### DISCUSSION TITLE
📣 Call for guest writers on daily.dev's blog

### DISCUSSION THREAD

**Discussion by nimrodkra** [dailydotdev/daily#230] [General]
# Join us as a guest blogger! ✏️
## It is fun, satisfying, and rewarding 🤗

![funny GIF](https://media.giphy.com/media/AGGz7y0rCYxdS/giphy.gif)

daily.dev is more than a browser extension. It is a thriving community of developers getting together around discovering and exploring high-quality dev news. By writing a guest post on our blog, you will be helping +75,000 developers from all over the world. 

## Why should you submit a story? 🎁
Here are some of the benefits of submitting a story to daily.dev/blog:
* We will publish your story on [daily.dev official blog](https://daily.dev/blog).
* Your story will be **featured on the daily.dev browser extension feed** (😱). It means that **+75K developers** from all over the world will have the chance to be exposed to your story every time they open a new tab on their browsers.
* We will **feature your story on our Weekly Recap newsletter** sent every Monday to tens of thousands of developers in our community.
* We will share your story on daily.dev’s Twitter, Facebook, Instagram, and LinkedIn social media official accounts. 

# How to submit a guest post? 🔥
We accept guest post submissions through an online form. 
Interested? Check out the complete guide in [this link](https://daily.dev/submit-a-story).

**Comment by Kvaibhav01** [comment]
As a guest blogger for daily.dev myself, I can assure you that the team is really supportive, they will share your piece anywhere they can for maximum reach. 
 
If you have written a quality article, then surely this is a good way to go. 🚀

**Reply by idoshamun** [reply]
Thank you so much for supporting us along the way 🙏

**Reply by killshot13** [reply]
I, too, can attest to this statement.

**Comment by rebelchris** [comment]
I agree very helpful in getting your article up to the highest standards.
Thanks, guys for having this option.

Looking forward to submitting some more in 2021

**Reply by idoshamun** [reply]
Thank you, Chris 🙌

**Comment by esau-morais** [comment]
My first experience writing to a blog was actually just because of Daily.dev (not forgetting about how much @nimrodkra and @idoshamun helped) and there's nothing else better than writing about your experience and sharing it with everybody.

**Reply by nimrodkra** [reply]
Thanks, Esau! It's our pleasure.

**Comment by J3ffJessie** [comment]
Are we able to submit writing that was published elsewhere or only original content? I post a series on Dev.to and wanted to expose it to daily.dev if possible.

**Reply by nimrodkra** [reply]
Sorry but we accept only original content that wasn't published elsewhere. There are plenty of benefits in terms of the traffic you can get by submitting a post on daily.dev which doesn't make sense for us to open it for republishing. 

Generally, if you wish to have your content picked up by daily dev there are several ways:
1. **Publish guest articles on well-known publications** that are already included in the source list of daily.dev. You can explore the list using the sidebar in the app itself. There are plenty of options out there.
2. **Publish articles on developer blogging platforms** such as Hacker Noon, Hashnode, or DEV. If your article will gain enough traction on those platforms, there's a high chance it will be picked up by daily.dev as well.
3. **Submit a new and original article to daily.dev/blog** which I covered in the post above.

**Reply by shadowtime2000** [reply]
@nimrodkra Would combining a series you have previously done into one long post be allowed? Or would that be considered unoriginal content?

**Reply by nimrodkra** [reply]
Combining existing content doesn't qualify under original content. Sorry.

**Comment by shadowtime2000** [comment]
Could I write a one line call to action at the end of the post like, "If you liked this article I would recommend you check out other articles of mine at [my blog]" or something like that?

**Reply by nimrodkra** [reply]
Sorry but we usually take those kinds of CTAs down since it's overly promotional. I would suggest submitting a post related to another topic you've written about in the past on your personal blog. That way, you'd be able to add a link to it, referring readers to this specific article from your blog for further information. It would simply be more organic.

**Reply by shadowtime2000** [reply]
That makes sense.

**Comment by shadowtime2000** [comment]
I noticed on the form it asked for a full name - are anonymous people still allowed? What is the policy around that?

**Reply by nimrodkra** [reply]
We require a real name as it increases the authenticity of the content. That is an important factor for any publication as we want our readers to know who wrote it and who's accountable for the content.

**Reply by shadowtime2000** [reply]
@nimrodkra By real name do you mean human sounding name or the name that you got when you are born? So like are pen names allowed in that case?

**Reply by nimrodkra** [reply]
Obviously, we have no way to verify what is the name people were born with. We trust our writers to have integrity by letting their readers know who they are and how they can reach out if there's a question or comment they want to make. 

To your question - we check every guest post based on all the details that the author submitted, including the background details such as name, Twitter account, and profile image. The content itself + the background details serve us to decide whether we accept the post or not. So I guess my answer is "it depends". Theoretically, we might accept a pen name, if it was already used by that author in a wide range of articles she/he published online beforehand.
